lib/elixir/lib/node.ex

```diff
@@ -72,7 +72,8 @@ defmodule Node do
   The result returned when the argument is a list, is the list of nodes
   satisfying the disjunction(s) of the list elements.
 
-  See http://www.erlang.org/doc/man/erlang.html#nodes-1 for more info.
+  For more information, see
+  [`:erlang.nodes/1`](http://www.erlang.org/doc/man/erlang.html#nodes-1).
```

Comment:
actually, nevermind. this is good to go. even more because those links are an important part of the docs and not just an addition.